index.js

```diff
@@ -1,46 +1,71 @@
+require("./messages");
+var steam = require("steam");
+
+const DOTA_APP_ID = 570;
+
 var EventEmitter = require('events').EventEmitter,
     fs = require("fs"),
+    path = require("path"),
     util = require("util"),
     bignumber = require("bignumber.js"),
-    Schema = require('protobuf').Schema,
-    base_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/base_gcmessages.desc")),
-    gcsdk_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/gcsdk_gcmessages.desc")),
-    dota_gcmessages = new Schema(fs.readFileSync(__dirname + "/generated/dota_gcmessages.desc")),
-    protoMask = 0x80000000,
+    ProtoBuf = require('protobufjs'),
     Dota2 = exports;
 
+ProtoBuf.convertFieldsToCamelCase = true;
```

Comment:
I wouldn't worry too much about the effects of doing big changes. We have major upstream changes and a whole new dota2 client. Now is the time to make big changes. We can keep a legacy branch, but now is the best time to break things and for users to refactor their stuff. Just need to keep track of the read-me